lib/new.js

```diff
@@ -46,53 +44,60 @@ const isRepoEmpty = (dir) => {
 const npmInstall = () => {
   logger.info('Running npm install');
   const npmProcess = spawn('npm', ['install'], { cwd: settings.path, stdio: 'inherit' });
-  npmProcess.on('exit', function () {
+  npmProcess.on('exit', () => {
     logger.info('SPA %s created in directory %s', settings.spa, settings.name);
     logger.info('Change into that directory and run "skyux serve" to begin.');
   });
+
+  return Promise.resolve();
```

Comment:
Is there any concern about immediately resolving the promise?  Perhaps it would be safer to return a new promise and resolve it in the exit handler?